ksql-engine/src/main/java/io/confluent/ksql/schema/registry/KsqlSchemaRegistryClientFactory.java

```diff
@@ -34,19 +34,25 @@
   private final Supplier<RestService> serviceSupplier;
   private final Map<String, Object> schemaRegistryClientConfigs;
   private final SchemaRegistryClientFactory schemaRegistryClientFactory;
+  private final Map<String, String> schemaRegistryHttpHeaders;
 
   interface SchemaRegistryClientFactory {
     CachedSchemaRegistryClient create(RestService service,
                                       int identityMapCapacity,
-                                      Map<String, Object> clientConfigs);
+                                      Map<String, Object> clientConfigs,
+                                      Map<String, String> httpHeaders);
```

Comment:
nit: use the same variable name (httpHeaders or schemaRegistryHttpHeaders)

Reply:
Done